exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -172,22 +174,43 @@ func (prwe *prwExporter) PushMetrics(ctx context.Context, md pmetric.Metrics) er
 	case <-prwe.closeChan:
 		return errors.New("shutdown has been called")
 	default:
+		var tsMap map[string]*prompb.TimeSeries
```

Comment:
you don't need to predefine all of these anymore

Reply:
fixed